javascript-timers/interval.js

```diff
@@ -0,0 +1,16 @@
+var $countdown = document.querySelector('h1');
+
+
+function countdown() {
+  if ($countdown.textContent === '4') {
+    $countdown.textContent = '3';
+  } else if ($countdown.textContent === '3'){
+    $countdown.textContent = '2';
+  } else if ($countdown.textContent === '2'){
+    $countdown.textContent = '1';
+  } else if ($countdown.textContent === '1'){
```

Comment:
What if the timer started from 1000? You should try to find a way to do this that doesn't require a conditional for each number.